<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки программы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Порт</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="portBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="addressBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="autoConnectCheckBox">
     <property name="text">
      <string>Автоподключение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="baudBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="hexAddressLabel">
     <property name="text">
      <string>Адрес (hex): 57h</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Бод рейт</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Адрес устройства</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
